### PR TITLE
Update Vista3d eval and add zero shot descriptions

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Fixes # .

### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Review by KumoLiu — COMMENTED** [review]
Thanks for the update, looks good to me.
I have another question regarding the extra metric in the training config, seems it'n not used, do we need include it here? Thanks.
https://github.com/Project-MONAI/model-zoo/blob/dev/models/vista3d/configs/train.json#L355C10-L355C28
https://github.com/Project-MONAI/model-zoo/blob/dev/models/vista3d/configs/train.json#L369

**Comment by yiheng-wang-nv** [comment]
> Thanks for the update, looks good to me. I have another question regarding the extra metric in the training config, seems it'n not used, do we need include it here? Thanks. https://github.com/Project-MONAI/model-zoo/blob/dev/models/vista3d/configs/train.json#L355C10-L355C28 https://github.com/Project-MONAI/model-zoo/blob/dev/models/vista3d/configs/train.json#L369

Hi @KumoLiu , thanks for the question. There are some configs that are not used, like `additional_metrics` as you mentioned. I'm not sure why we include it in the config file. Hi @heyufan1995 , could you provide some information here? Thanks!

**Comment by heyufan1995** [comment]
> > Thanks for the update, looks good to me. I have another question regarding the extra metric in the training config, seems it'n not used, do we need include it here? Thanks. https://github.com/Project-MONAI/model-zoo/blob/dev/models/vista3d/configs/train.json#L355C10-L355C28 https://github.com/Project-MONAI/model-zoo/blob/dev/models/vista3d/configs/train.json#L369
> 
> Hi @KumoLiu , thanks for the question. There are some configs that are not used, like `additional_metrics` as you mentioned. I'm not sure why we include it in the config file. Hi @heyufan1995 , could you provide some information here? Thanks!

Seems like it was not used. It was created by Wenqi @wyli

**Review by heyufan1995 — APPROVED** [review]
